### DISCUSSION TITLE
How to change the title and description?

### DISCUSSION THREAD

**Discussion by iYonga** [DirectoryLister/DirectoryLister#608] [Help & Support] [answered]
I scoured through all files, but can't find it :( I don't want "Directory Lister" to be seen all over my directory.

**Comment by PHLAK** [comment, marked as answer]
The title and description can be changed with the [`SITE_TITLE`](https://docs.directorylister.com/configuration/app-config-reference#site_title) and [`META_DESCRIPTION`](https://docs.directorylister.com/configuration/app-config-reference#meta_description) environment variables respectively.

**Reply by iYonga** [reply]
![image](https://user-images.githubusercontent.com/30579752/114281269-25f2f600-9a46-11eb-9d63-d1dd3c694237.png)

Isn't this how I should be doing it? I cleared cache too but it is just stock with the old one. Visu.pub as a website is currently active too, FYI.

**Comment by PHLAK** [comment]
The easiest way to modify configuration options is through the [.env file](https://docs.directorylister.com/configuration/configuration-overview#the-env-file). You can accomplish this by copying the provided `.env.example` file to `.env` and modifying or adding configuration variables there. For example...

    cp .env.example .env

Then add the  `SITE_TITLE` and `META_DESCRIPTION` variables.

```
SITE_TITLE="Test Title; Please Ignore"
META_DESCRIPTION="Some new description"
```

**Reply by iYonga** [reply]
shucks! thanks! I thought .env and the other config types were separate things.

**Reply by PHLAK** [reply]
The `.env` file is a simplified configuration file. The files in `app/config` read the variables set in `.env`. It is recommended to use the `.env` file unless you need more advanced control over your configuration.
